Makefile

```diff
@@ -728,29 +821,27 @@ hostname.o: \
 compile hostname.c substdio.h subfd.h substdio.h readwrite.h exit.h
 	./compile hostname.c
 
-idedit: \
-load idedit.o strerr.a substdio.a error.a str.a fs.a wait.a open.a \
-seek.a
-	./load idedit strerr.a substdio.a error.a str.a fs.a \
-	wait.a open.a seek.a 
-
-idedit.o: \
-compile idedit.c readwrite.h exit.h scan.h fmt.h strerr.h open.h \
-seek.h fork.h
-	./compile idedit.c
-
 install: \
-load install.o fifo.o hier.o auto_qmail.o auto_split.o auto_uids.o \
-strerr.a substdio.a open.a error.a str.a fs.a
+load install.o fifo.o hier.o auto_qmail.o auto_split.o uid.o gid.o \
+auto_usera.o auto_uida.o auto_usero.o auto_uido.o auto_userp.o auto_uidp.o \
+auto_userq.o auto_uidq.o auto_userr.o auto_uidr.o auto_users.o auto_uids.o \
+auto_groupq.o auto_gidq.o strerr.a substdio.a open.a error.a str.a fs.a
 	./load install fifo.o hier.o auto_qmail.o auto_split.o \
-	auto_uids.o strerr.a substdio.a open.a error.a str.a fs.a 
+	uid.o gid.o auto_usera.o auto_uida.o auto_usero.o auto_uido.o \
+	auto_userp.o auto_uidp.o auto_userq.o auto_uidq.o auto_userr.o \
+	auto_uidr.o auto_users.o auto_uids.o auto_groupq.o auto_gidq.o \
+	strerr.a substdio.a open.a error.a str.a fs.a
 
 install-big: \
-load install-big.o fifo.o install.o auto_qmail.o auto_split.o \
-auto_uids.o strerr.a substdio.a open.a error.a str.a fs.a
+load install-big.o fifo.o install.o auto_qmail.o auto_split.o uid.o gid.o \
+auto_usera.o auto_uida.o auto_usero.o auto_uido.o auto_userp.o auto_uidp.o \
+auto_userq.o auto_uidq.o auto_userr.o auto_uidr.o auto_users.o auto_uids.o \
+auto_groupq.o auto_gidq.o strerr.a substdio.a open.a error.a str.a fs.a
 	./load install-big fifo.o install.o auto_qmail.o \
-	auto_split.o auto_uids.o strerr.a substdio.a open.a error.a \
-	str.a fs.a 
+	auto_split.o uid.o gid.o auto_usera.o auto_uida.o auto_usero.o \
+	auto_uido.o auto_userp.o auto_uidp.o auto_userq.o auto_uidq.o \
+	auto_userr.o auto_uidr.o auto_users.o auto_uids.o auto_groupq.o \
+	auto_gidq.o strerr.a substdio.a open.a error.a str.a fs.a
```

Comment:
This helps me lean toward whole-horse :-)

Reply:
In that event there is PR #22 removing the shar target and precompiled var-qmail package support.  If that went in before this patch it would remove these lines--but as @DerDakon notes it conceptually fits best right after the first patch in this series, where the smaller idedit removal is now.

I can pull these commits apart again, rehydrating PR #11 to submit the first of these commits, or I can pull in PR #22 to this (already large, I'm afraid) patch series, and I think that gets everyone's needs met.